components/eventsrv/Cargo.toml

```diff
@@ -10,8 +10,11 @@ workspace = "../../"
 [dependencies]
 byteorder = "*"
 env_logger = "*"
+habitat_core = { path = "../core" }
```

Comment:
This pulls in quite a few dependencies that we don't really need so we should maybe revisit what is in core in the future or split out some of the useful modules here into something more appropriate. For example, we don't need libarchive at all but we totally want the ConfigFile trait in this crate